py/tests/algorithms/anomaly_likelihood_test.py

```diff
@@ -415,7 +419,7 @@ def testSampleDistribution(self):
          "name": "normal",
          "stdev": math.sqrt(0.1),
          "variance": 0.1}
-    samples = _sampleDistribution(p, 1000)
+    samples = _sampleDistribution(p, 1000, seed=Random(self.GLOBAL_TEST_SEED).getUInt32())
```

Comment:
this might be overkill, seed=GLOBAL_TEST_SEED should be enough

Reply:
ok right, it is now corrected